### DISCUSSION TITLE
Output or manifest

### DISCUSSION THREAD

**Discussion by sestegra** [vmware/packer-examples-for-vsphere#51] [Ideas]
Hello,

In the current `README.md`, Packer manifests are in `manifests` folder.
> `manifests` - manifests created after the completion of each build.

However, the current manifests are stored (and not ignored by GIt) in `output` path relative to the running build.

I suggest the following:
- move manifests to `manifests` folder
- Git ignore `manifests` folder

To do so, use Packer `path` variable to output manifest in `<ROOT>/manifests` folder.

What do you think about it?

Kind regards,
Stéphane

**Comment by tenthirtyam** [comment]
In a previous release I had these going to `/manifests` and then moved them to the `output`of the build.

I'll reconsider moving these back to original path with a `.gitkeep` for the folder but a `.gitignore ` for the contents.

Ryan

**Comment by tenthirtyam** [comment]
Please clone to an enhancement request and I'll plan to address this refactor.

Ryan

**Comment by tenthirtyam** [comment]
Addressed in https://github.com/rainpole/packer-vsphere/pull/66.

Ryan
